src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import Movie from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  card: Movie;
+}
+
+const MovieCard: React.FC<Props> = ({ card }) => (
+  <>
+    <div className="card">
```

Comment:
```suggestion
    <div className="card">
```
Why do you need React fragment here? You have card container instead
![image](https://user-images.githubusercontent.com/96409563/225838782-5927c0c6-5473-4c3e-8ac8-051baf4a7327.png)